titan-server/src/api.rs

```diff
@@ -1,32 +1,80 @@
 use std::io::Cursor;
 use diesel::result::Error as DieselError;
-use rocket::response::Responder;
+use rocket::response::{Responder, Content};
```

Comment:
Running this through `clippy`, it doesn't appear that `Content` is a used import.